cmd/swarmctl/service/flagparser/flags.go

```diff
@@ -35,7 +35,11 @@ func AddServiceFlags(flags *pflag.FlagSet) {
 
 	flags.Uint64("update-parallelism", 0, "task update parallelism (0 = all at once)")
 	flags.String("update-delay", "0s", "delay between task updates (0s = none)")
-	flags.String("update-on-failure", "pause", "action on failure during update (pause|continue)")
+	flags.String("update-on-failure", "pause", "action on failure during update (pause|continue|rollback)")
+
+	flags.Uint64("rollback-parallelism", 0, "task update parallelism during rollback (0 = all at once)")
```

Comment:
I think this should have the same default as `update-parallelism`. I know that in Docker `update-parallelism` defaults to 1. We should probably fix it to do the same here. It's not really easy though - under our current system, the only way to do that is to hardcode those defaults in the client. I have a proposal for doing this better in #1744.